src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -108,8 +113,208 @@ public void getObjects() throws Exception {
+
+        queryParams.put(ObjectPersistenceService.TYPE, "type z");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 0);
+
+        queryParams.put(ObjectPersistenceService.TYPE, "ype");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 0);
+
+        queryParams.remove(ObjectPersistenceService.TYPE);
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "moniker");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 3);
+
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "moniker o");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 1);
+
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "moniker t");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 2);
+
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "moniker three");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 1);
+
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "moniker z");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 0);
+
+        queryParams.put(ObjectPersistenceService.MONIKER_LIKE, "oniker");
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 0);
+
+    }
+
+    @Test
+    public void findByQueryParametersLastModified() throws Exception
+    {
+        final UUID accountUuid = UuidUtil.getNewUuid();
+        final String accountUrn = UuidUtil.getAccountUrnFromUuid(accountUuid);
+        Map<String, Object> queryParams = new HashMap<>();
+
+        Long firstDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
+
+        ObjectEntity firstObject = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnLastModTest1").name("last mod test 1").type("anythingIsFine").build();
+        this.objectRepository.save(firstObject);
+
+        Long secondDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
+
+        ObjectEntity secondObject = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnLastModTest2").name("last mod test 2").type("anythingIsFine").build();
+        this.objectRepository.save(secondObject);
+
+        Long thirdDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
```

Comment:
There has to be a better way.  Introducing `Thread.sleep()` creates unwanted delays in UT completion.  We should be able to populate the integration test database with canned data and query it.